scripts/test-all.sh

```diff
@@ -95,8 +95,9 @@ if rustup toolchain list | grep -q nightly; then
     cargo +nightly test --features lightbeam
 
     # Also run wasmtime-py and wasmtime-rust's tests.
+    # Temporarily disable wasmtime-py due to https://github.com/bytecodealliance/wasmtime/issues/468
+    #  --package wasmtime-py
     RUST_BACKTRACE=1 cargo +nightly test \
-      --package wasmtime-py \
```

Comment:
How come this is needed here?

Reply:
See the comment just above; there's currently a bug affecting testing of wasmtime-py.